cmd/gke-exec-auth-plugin/request.go

```diff
@@ -73,6 +76,9 @@ func kubeEnvToConfig(kubeEnv string) (*rest.Config, error) {
 			key = string(keyBytes)
 		case strings.HasPrefix(l, kubeEnvMaster):
 			master = strings.TrimPrefix(l, kubeEnvMaster)
+		case strings.HasPrefix(l, kubeEnvCaFilePath):
+			caFilePath = strings.TrimPrefix(l, kubeEnvCaFilePath)
+			klog.Info("Using CA file path from kube-env: %v", caFilePath)
```

Comment:
`klog.Infof` and use `%q` instead of `%v`

Reply:
Done. I just spent a couple minutes looking for klog / glog / golang log/fmt documentation and found nothing very useful :/